package.json

```diff
@@ -14,8 +14,9 @@
     },
     "main": "index.tsx",
     "scripts": {
-        "start": "webpack-dev-server --mode=development",
-        "build": "webpack --mode=production",
+        "start": "webpack-dev-server --mode=development --env backend=local",
+        "startNectar": "webpack-dev-server --mode=development --env backend=online",
```

Comment:
Should the `startNectar` command be setting the webpack mode to development or production? If this command would actually be used by Nectar then it should probably be set to production right?

Reply:
The way I envisioned it is `startNectar` can be used if you are developing but aren't making any changes to the backend, hence it still has to run in dev mode. When you want a production build and online backend, that is covered by the `build` command.